lib/db.c

```diff
@@ -11162,10 +11163,11 @@ grn_ctx_at(grn_ctx *ctx, grn_id id)
                 "<%u>(<%.*s>)",
                 id,
                 name_size, name);
-            grn_db_value_unlock(ctx, id, vp);
+          }
+          grn_db_value_unlock(ctx, id, vp); /* This sentence is always executed because there is no condition. */
```

Comment:
The comment is needless.
It's just for explaining this to you.

Reply:
Oh...
I've removed it.